docs/source/install/instructions/source.rst

```diff
@@ -75,8 +83,16 @@ The syntax is still very similar:
    make
    make install
 
-That's all!
-Continue with the following section to build our dependencies.
+One can usually pass further options with ``--with-something=VALUE`` or ``--enable-thing`` to ``configure``.
+See ``configure --help`` when installing an *autotools* project.
+
+That is all on the theory of compiling projects!
+
+Now Start
+^^^^^^^^^
+
+You now know all basics to install from source.
```

Comment:
I think should be `all the basics`

Reply:
yeah, was thinking about the article and it's actually not necessary but... ok.